.github/workflows/stale.yml

```diff
@@ -0,0 +1,22 @@
+name: Mark and Close Stale Issues
+
+on:
+  schedule:
+    # Every Monday at 1am
```

Comment:
nit: Do we know the workflow timezone? If so, could we specify that in the comment above?

Reply:
addressed